### DISCUSSION TITLE
无法记账

### DISCUSSION THREAD

**Discussion by DreamSWL** [AutoAccountingOrg/AutoAccounting#307] [求助提问]
自动记账里面可以记账，账单里面也有。付款以后不会弹窗。也不会同步到账本里面。每一个账单都要手动点开以后再点保存才会出现在钱迹里。悬浮窗和所有的权限都给了。旧版3.3.4可以正常弹窗，新版不能正常弹窗不记账，已打钱迹补丁，已检查已开启，MIUI13官改系统

**Comment by AnkioTomas** [comment]
beta10的bug
